source/features/hidden-review-comments-indicator.css

```diff
@@ -30,7 +30,7 @@
 
 .rgh-comments-indicator button {
 	position: absolute;
-	right: 100%;
+	right: 90%;
```

Comment:
Why? It seems too close now

Reply:
Maybe it's ok

Reply:
The Comments icon is cut in 1800px with `100%` but we can try :)